src/components/mdRatingBar/mdRatingBar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="md-rating-bar" :class="[themeClass]" :disabled="disabled">
+    <div class="back-stars" :disabled="disabled">
+      <md-icon v-for="i in numStars" :key="i"
+               @mouseover.native="hoverStars"
+               @click.native="clickStars"
+               @mouseout.native="onMouseOut">{{ backIcon }}</md-icon>
+    </div>
+    <div class="front-stars" :style="frontStarsStyle">
+      <md-icon v-for="i in numStars"
+               @mouseover.native="hoverStars"
+               @click.native="clickStars"
+               @mouseout.native="onMouseOut">{{ frontIcon }}</md-icon>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" src="./mdRatingBar.scss"></style>
+
+<script>
+  import theme from '../../core/components/mdTheme/mixin';
+
+  let iconSize = 24;//size of each icon from rating bar in pixels
+
+  export default {
+    props: {
+      numStars: {
+        type: Number,
+        default: 5
+      },
+      disabled: Boolean,
+      value: {
+        type: Number,
+        validator: (value) => {
+          return value >= 0 && value <= 1;
```

Comment:
That's interesting from an API standpoint. While I can see why you would use a number between 0 and 1, it could also be a number between 0 and 100. or 0 and `md-num-stars`.

_Personally_, I would actually expect it to be [0, `md-num-stars`]. That way, I can directly say that I have `3.5` our of 5 stars:

```html
<md-rating-bar v-model="reviewAvg" :md-num-stars="5" class="md-primary"></md-rating-bar>
```

```js
{
    ...,
    computed: {
        reviewAvg(){ return _.sum(this.reviews) / this.reviews.length }
    },
    ...
}
```